modules/channel-smooch/src/config.ts

```diff
@@ -22,4 +22,12 @@ export interface Config {
    * @default 24h
    */
   chatUserAuthDuration: string
+
+  /**
+   * Enables forwarding of raw smooch payloads for the provided payload types.
+   * Add the "smooch-" prefix to forward events specific to the smooch channel
+   * The raw payload is accessible in event.payload.channel.smooch
+   * @default []
+   */
+  forwardRawPayloads: string[]
```

Comment:
Can you add an example here!

Reply:
If I understand properly, you would add something like [ 'smooch-video' ] as forwardRawPayloads in order for the payload to be accessible through payload.channel.smooch.message ?

Reply:
Yes I made this so it doesn't break if we eventually add the video payload ourselves. Otherwise what's being sent out as event.type === `video` would change once we added support for it.

So instead if you want to forward a message type that we don't support yet, you have to watch for event with event.type === 'smooch-video'